lib/taurus/qt/qtgui/display/tauruslabel.py

```diff
@@ -419,9 +419,17 @@ def getPermanentText(self):
         return self._permanentText
 
     def setPermanentText(self, text):
```

Comment:
I think that setPermanentText() should be renamed to setText()
Note that setPermanentText has not been in any released version of Taurus (and therefore we should not worry about compatibility)

Reply:
`permanentText` is a property with setter and getter `getPermanentText and setPermanentText` 
Do you propose to change this API? or just change setPermanentText by setText?

Reply:
good point... actually if we want to provide allow the config property to work, we need to keep it... 
So I change my proposal to renaming `setPermanentText` to `_setPermanentText` (so that there is only 1 public accessor).

Reply:
Done!